packages/editor-ui/src/views/CredentialsView.vue

```diff
@@ -76,17 +77,18 @@ const needsSetup = (data: string | undefined): boolean => {
 
 const allCredentials = computed<Resource[]>(() =>
 	credentialsStore.allCredentials.map((credential) => ({
+		resourceType: 'credentials',
```

Comment:
nitpick / optional - `credential` instead of `credentials` might be more clearer

Reply:
Yup, agree

Reply:
Now looking into this, a bunch of `i18n` keys and some other things is `ResourceListLayout` are relying on plural here. So, if it's an optional nit, I would like to keep it this way

Reply:
Yes, optional.